tests/components/tracked-controls.test.js

```diff
@@ -465,3 +568,43 @@ function toQuaternion (x, y, z) {
     return quaternion.toArray();
   })();
 }
+
+/**
+ * Given a button event emit call that has been spied, verify all of the
+ * data matches.
+ *
+ * @param {object} eventCall - The spied call to emit.
+ * @param {string} eventName - The expected name of the event.
+ * @param {number} eventId - The button id firing the event.
+ * @param {object} eventState - The full event state.
+ */
+function assertButtonEvent (eventCall, eventName, eventId, eventState) {
+  assert.equal(eventCall.args[0], eventName);
+  assert.deepEqual(eventCall.args[1].id, eventId);
```

Comment:
Should this just be `assert.equal(eventCall.args[1].id, eventId);` ?

Reply:
yes.